main.nf

```diff
@@ -1142,6 +1144,13 @@ def checkFileExtension(it, extension) {
   }
 }
 
+def checkGenome() {
+  if (!(params.genome)) {
+    return false
+  }
+  return params.genome in ['GRCh37', 'GRCh38'] ? true : false
```

Comment:
the `in` operator already returns a boolean. no need to check again whether it is `true` or `false`. Use just
```
return params.genome in ['GRCh37', 'GRCh38']
```
And I would also not check this at all. And I would actually just check `params.genome in params.genomes`. No need to artifically restrict this. We are going to have a custom version of GRCh38, for example, and then it would be nice to just name it `GRCh38b` or something.